regression/global_enkf.sh

```diff
@@ -76,14 +76,14 @@ export use_correlated_oberrs=${use_correlated_oberrs:-".false."}
 if [ $USE_CORRELATED_OBERRS == "YES" ]; then
     export use_correlated_oberrs=".true."
 fi
-export imp_physics=${imp_physics:-"11"}
+export imp_physics=${imp_physics:-"8"}
 export lupp=${lupp:-".true."}
 export corrlength=${corrlength:-1250}
 export lnsigcutoff=${lnsigcutoff:-2.75}
 export analpertwt=${analpertwt:-0.85}
 export readin_localization_enkf=".false."
```

Comment:
Do you know why this line was here separately rather than changing the default?  Is it still needed?

Reply:
The line I'm referring to is Line 84: export readin_localization_enkf=".false."

Reply:
I assume the namelist variables in this script trace back to when this script was used as a stand-alone script to run `enkf.x`.  This script is now used for ctests.  Remove the following lines from `global_enkf.sh`
```
-export imp_physics=${imp_physics:-"8"}
-export readin_localization_enkf=${readin_localization_enkf:-".false."}
-export reducedgrid=${reducedgrid:-".false."}
```
Also update global_enkf section of `regression_namelists.sh` to explicitly set `imp_physics=8` as is done for global_4denvar.

Done at 11a80d6.